kola/tests/kubeadm/kubeadm.go

```diff
@@ -75,6 +75,38 @@ var (
 	// testConfig holds params for various kubernetes releases
 	// and the nested params are used to render script templates
 	testConfig = map[string]map[string]interface{}{
+		"v1.27.2": map[string]interface{}{
+			"MinMajorVersion": 3374,
+			// from https://github.com/flannel-io/flannel/releases
+			"FlannelVersion": "v0.22.0",
+			// from https://github.com/cilium/cilium/releases
+			"CiliumVersion": "1.12.5",
+			// from https://github.com/cilium/cilium-cli/releases
+			"CiliumCLIVersion": "v0.12.12",
```

Comment:
I can see that latest Cilium is at [1.14.0-snapshot.3](https://github.com/cilium/cilium/releases/tag/1.14.0-snapshot.3) and CLI at [v0.14.7](https://github.com/cilium/cilium-cli/releases/tag/v0.14.7). Any reason for not using those?